doc/HandlerConfiguration.md

```diff
@@ -1,92 +1,170 @@
-# Declarative proxy handler definition
-- Provide a clear spec of the request flow for a given entry point
-- Abstract common operations
-- Make it easy to port to another environment later
+# Declarative request handler definition
```

Comment:
@d00rman haha, this is because you wrote it (https://phabricator.wikimedia.org/T105975#1499210)